### ISSUE TITLE
Can it be installed by submodules?

### ISSUE THREAD

**Issue by mig82** [josdejong/mathjs#1784]
I'd just need the definition for Greatest Common Divisor. Is it possible to install a submodule of MathJS?

**Comment by josdejong** [comment]
Yes, if you're using ES6 imports you can just import the functions you need. The regular functions like the following are relatively large since they include BigNumber, Matrix, etc:

```js
import { gcd } from 'mathjs'
```

If you only need a `number` implementation I think the following will work for you:

```js
import { gcd } from 'mathjs/number'
```

For more info see:

https://mathjs.org/docs/getting_started.html
https://mathjs.org/docs/custom_bundling.html

**Comment by mig82** [comment]
Thank you @josdejong. I forgot to mention I'm using this in Node, so I was looking for something like a partial installation —like `npm install mathjs.gcd`. The way Lodash does it. I take it that's not possible or you would have mentioned it.

**Comment by josdejong** [comment]
No, not really, though you can look in `/lib/plain/number/arithmetic` for example, it's quite small. 

Just curious: if you're using node.js, a larger file is normally not an issue like in the browser where bundles must be as small as possible. Why are you concerned with loading a small file only?

**Comment by nhammond101** [comment]
@josdejong my use case is lambda functions, so creating the smallest bundles is beneficial for speed

**Comment by josdejong** [comment]
Yes that makes sense Nick 👍

**Comment by mig82** [comment]
In my case I'm building a mobile app using Temenos (formerly Kony) Quantum and I'm just abusing NPM to install dependencies. So also, the smaller the codebase the better.

**Comment by josdejong** [comment]
Thanks for your feedback Miguelángel. For web and mobile, the bundle size matters a lot indeed, only for nodejs (server side) it's less important.
